assets/scripts/libraries/cookies-opt-in.js

```diff
@@ -374,7 +374,7 @@ var docCookies = {
         var cookieMessageInner = document.createElement('div');
         cookieMessageInner.classList.add('cookie-message__inner', 'govuk-width-container');
         cookieMessageInner.classList.add('site-container');
-        cookieMessageInner.innerHTML = '<div class="cookie-message__intro"><p>We use cookies to collect information about how you use crowncommercial.gov.uk. We use this information to make the website work as well as possible and improve government services.</p></div>';
+        cookieMessageInner.innerHTML = '<h2 class="govuk-heading-m">Cookies on crowncommercial.gov.uk</h2><div class="cookie-message__intro"><p>We use cookies to collect information about how you use crowncommercial.gov.uk.<br><br> We use this information to make the website work as well as possible and improve government services.</p></div>';
```

Comment:
It's likely we shouldn't be using H2 here due to accessibility. Header tags need to be nested correctly if I understand correctly. https://www.w3.org/WAI/WCAG21/Techniques/html/H42